api/models/post.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require("sequelize");
+const sequelize = require("../db/db.js");
+const User = require("./User.js");
+
+const Post = sequelize.define("posts", {
+  id: {
+    type: Sequelize.INTEGER(11),
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+
+  topic: Sequelize.STRING(250),
+  discription: Sequelize.STRING(400),
+  city: Sequelize.STRING(255),
```

Comment:
Can you make here strict such as` allowNull: false`

Reply:
done